client/verta/verta/_uac/_organization.py

```diff
@@ -0,0 +1,22 @@
+from ._role import Role
+from verta._protos.public.uac import GroupV2_pb2 as _Group
+from verta._protos.public.uac import RoleV2_pb2 as _Role
+class OrganizationV2:
+    def __init__(self, conn, org_id):
+        self.conn = conn
+        self.org_id = org_id
+
+    def get_groups(self):
+        response_groups = self.conn.make_proto_request(
+            "GET", "/api/v1/uac-proxy/organization/{}/groups".format(self.org_id)
```

Comment:
I'm not sure about this, but should we use `v2` instead of `v1`? 🤔 

```suggestion
            "GET", "/api/v1/uac-proxy/organization/{}/groups".format(self.org_id)
```

Reply:
I think v2 makes more sense, but need to change it in proto first: https://github.com/VertaAI/protos-all/pull/425